src/common/modules/Logger.js

```diff
@@ -13,6 +13,36 @@ const MESSAGE_LEVEL_NAME = Object.freeze({
     [MESSAGE_LEVEL.SUCCESS]: "SUCCESS"
 });
 
+/**
+ * Checks if value is a object.
+ *
+ * @function
+ * @param  {*} value
+ * @returns {boolean}
+ */
+function isObject(value){
+    const type = typeof value;
+    return value != null && (type === "object" || type === "function");
+}
+
+/**
+ * Copy nested objects to ensure a properly output.
+ *
+ * @function
+ * @param  {...*} args
```

Comment:
`{...*}` means can pass infinitely amounts of parameters. That's not true here. You actually do only pass an array.